web-wallet/src/lib/containers/StakeContract/StakeContract.svelte

```diff
@@ -161,9 +171,9 @@
           {minAllowedStake}
           on:operationChange
           on:suppressStakingNotice
-          rewards={stakeInfo.reward}
+          rewards={luxToDusk(stakeInfo.reward)}
           spendable={balance.shielded.spendable}
-          staked={stakeInfo.amount}
+          staked={stakeInfo.amount ? luxToDusk(stakeInfo.amount.total) : 0}
```

Comment:
If the stakeInfo.amount is missing, it's like an error state, no? Or still loading?

Reply:
No, it can be `null`